sdk/template/aztemplate/ci.yml

```
@@ -0,0 +1,19 @@
+# NOTE: Please refer to https://aka.ms/azsdk/engsys/ci-yaml before editing this file.
+trigger:
+  paths:
+    include:
+    - sdk/template/aztemplate/
+    - eng/
+
+pr:
```

```suggestion
pr:
  branches:
    include:
    - main
    - feature/*
    - hotfix/*
    - release/*
```

We also need to include the branch filters so we only trigger for branches that follow our guidelines.